templates/kanban.php

```diff
@@ -59,14 +59,34 @@
                             </div>
                             <div class="card-body">
                                 <p>
-                                    <?php
-                                    echo($tasks[$arr]['due_date']);
-                                    ?>
+                                    <?=($tasks[$arr]['due_date']);?>
                                 </p>
                                 <a href="#" class="btn btn-tool">
                                     <i class="fas fa-file"></i>
                                 </a>
-                                <small class="badge badge-danger"><i class="far fa-clock"></i> 2 mins</small>
+                                <?php if  ($tasks[$arr]['due_date'] !== null): ?>
+                                    <?php
+                                    $data_time = strtotime($tasks[$arr]['due_date']);
+                                    $time_now = strtotime('now')+10800;
+                                    $difference = (($data_time-$time_now)/60/60);
+                                    $for_audit = ($data_time >= $time_now);
+                                    ?>
+
+                                    <small <?php if($for_audit):?> class="badge badge-success"<?php endif;?>
+                                            class="badge badge-danger"</i>
```

Comment:
Можна додати тільки назву класу під логіку
```suggestion
                                    <small class="badge badge-<?= $for_audit ? 'success' : 'danger' ?>">
```

Reply:
Тут до речі ломається верстка )